org.jacoco.core.test/src-java8/org/jacoco/core/test/validation/targets/BadCycleInterface.java

```diff
@@ -16,7 +16,7 @@
 	public interface Base {
 		static final Object BASE_CONST = new Child() {
 			{
-				Stubs.nop("base clinit"); // $line-1$
+				Stubs.nop("base clinit"); // $line-baseclinit$
```

Comment:
If execution order is relevant here it should be asserted. Should I try to record and assert execution order?